L7-JUnit Params, Mockito/L7-Zad.2-UpperCase/src/test/java/pl/babiarz/UpperCaseTest.java

```diff
@@ -0,0 +1,26 @@
+package pl.babiarz;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.NullAndEmptySource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class UpperCaseTest {
+
+    @ParameterizedTest
+    @ValueSource (strings = {"test", "TesT", "tEsT", "tESt"})
+    void shouldConvertTextToUpperCase (String text){
+        //given
+        //When
+        String upperCaseText = UpperCase.convertToUpperCase(text);
+        //Then
```

Comment:
piekło perfekcjonisty :P Jak już piszesz z małej litery "given" to wypadało by aby "When" i "Then" też było z małej